Three/include/CGAL/Three/Scene_item.h

```diff
@@ -285,6 +285,10 @@ public Q_SLOTS:
   void setPointsMode() {
     setRenderingMode(Points);
   }
+  //!Sets the RenderingMode to Points.
+  void setShadedPointsMode() {
+    setRenderingMode(ShadedPoints);
+  }
```

Comment:
Should not be switch to an API like `void setRenderingMode(int)`? That would avoid the need to change the `Scene_item.h` for a new rendering mode.